components/calendar/src/astronomy.rs

```diff
@@ -1368,37 +1373,30 @@ impl Astronomical {
 
     /// The parallax of the moon, meaning the difference in angle of the direction of the moon
     /// as measured from a given location and from the center of the Earth, in degrees.
-    ///
     /// Based on functions from _Calendrical Calculations_ by Reingold & Dershowitz,
     /// originally from _Astronomical Algorithms_ by Jean Meeus, 2nd edn., 1998.
     /// Lisp code reference: https://github.com/EdReingold/calendar-code2/blob/9afc1f3/calendar.l#L4619-L4628
-    #[allow(dead_code)] // TODO: Remove dead code tag after use
-    pub(crate) fn lunar_parallax(moment: Moment, location: Location) -> f64 {
-        let geo = Self::lunar_altitude(moment, location);
+    pub(crate) fn lunar_parallax(lunar_altitude_val: f64, moment: Moment) -> f64 {
         let cap_delta = Self::lunar_distance(moment);
         let alt = 6378140.0 / cap_delta;
-        let arg = alt * libm::cos(geo.to_radians());
+        let arg = alt * libm::cos(lunar_altitude_val.to_radians());
         arcsin_degrees(arg)
     }
 
-    /// Topocentric altitude of the moon, meaning the celestial altitude of the moon with the given
-    /// location as the origin, at a given moment, ignoring refraction.
-    ///
+    /// Topocentric altitude of the moon.
```

Comment:
nit: clarify in docs what the arguments to this function represent

Reply:
done :)